tests/manage/mcg/test_object_integrity.py

```diff
@@ -0,0 +1,73 @@
+import logging
+
+import boto3
+import pytest
+
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import ManageTest, tier1
+from ocs_ci.ocs import constants
+from tests.helpers import craft_s3_command
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.skipif(
```

Comment:
Should be changed to the `aws_platform_required` marker that will be added in PR718

Reply:
ack